numpy/core/_methods.py

```diff
@@ -187,16 +194,49 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     else:
         arrmean = arrmean.dtype.type(arrmean / rcount)
 
-    # Compute sum of squared deviations from mean
-    # Note that x may not be inexact and that we need it to be an array,
-    # not a scalar.
-    x = asanyarray(arr - arrmean)
-    if issubclass(arr.dtype.type, (nt.floating, nt.integer)):
-        x = um.multiply(x, x, out=x)
-    else:
-        x = um.multiply(x, um.conjugate(x), out=x).real
+    if issubclass(arr.dtype.type, (nt.floating, nt.complexfloating)):
+        # Compute the sum of (arr - m) and (arr - m)**2 over the last axis
+        if axis is None:
+            axis = tuple(range(arr.ndim))
+        elif isinstance(axis, int):
+            axis = (axis, )
+        else:
+            axis = tuple(sorted(axis))
+
+        dim_order = [i for i in range(arr.ndim) if i != axis[-1]] + [axis[-1]]
+        S1, S2 = umt.sumsquarediff(
+            arr.transpose(*dim_order),
+            arrmean.transpose(*dim_order),
+            keepdims=keepdims,
+            dtype=dtype
+        )
```

Comment:
I'm pretty sure you can pass `axis=axis[-1]` here and avoid needing to do the transpose

Reply:
Sadly no, because of the signature of the function
`TypeError: sumsquarediff: axis can only be used with a single shared core dimension, not with the 2 distinct ones implied by signature (i),(1)->(),()`. Is there broadcast now for gufunc ?

Reply:
`axes=[(axis[-1],), (axis[-1],), (), ()]` would work I think, but I forget the syntax alread

Reply:
> Is there broadcast now for gufunc ?

You could always do the broadcast yourself, with `arrmean = np.broadcast_to(arrmean, arr.shape)` - then declare your gufunc as `(i),(i)->(),()`. It would be marginally slower since there'd be two loop indices in the loop (one incrementing by zero).

Reply:
Can I access `broadcast_to` from `numpy/core` ?